src/DependencyInjection/Configuration.php

```diff
@@ -23,8 +23,9 @@ final class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('sulu_sylius_producer');
+        $treeBuilder = new TreeBuilder('sulu_sylius_producer');
+        // BC layer for symfony/config < 4.2
```

Comment:
why do we need to handle `symfony/config < 4.2`? the `composer.json` uses `"symfony/config": "^4.4|^5.0"`